lighthouse-core/gather/gatherers/offline.js

```diff
@@ -34,4 +36,8 @@ class Offline extends Gatherer {
   }
 }
 
+function cleanHash(url) {
+  return url.href.removeURLFragment();
```

Comment:
I'm not familiar with`removeURLFragment`. Where does it come from?

If it's not part of a spec, `url.origin + url.pathname + url.search` would be clear.